appserver/tests/tck/rest/pom.xml

```diff
@@ -0,0 +1,305 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (c) 2021, 20202 Contributors to the Eclipse Foundation. All rights reserved.
+
+    This program and the accompanying materials are made available under the
+    terms of the Eclipse Public License v. 2.0, which is available at
+    http://www.eclipse.org/legal/epl-2.0.
+
+    This Source Code may also be made available under the following Secondary
+    Licenses when the conditions for such availability set forth in the
+    Eclipse Public License v. 2.0 are satisfied: GNU General Public License,
+    version 2 with the GNU Classpath Exception, which is available at
+    https://www.gnu.org/software/classpath/license.html.
+
+    SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+
+-->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.glassfish.main.tests.tck</groupId>
+        <artifactId>tck</artifactId>
+        <version>6.2.4-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>glassfish-external-tck-rest</artifactId>
+    <packaging>pom</packaging>
+
+    <name>TCK: REST</name>
+
+    <properties>
```

Comment:
migrated JAXRS TCK(https://github.com/eclipse-ee4j/jaxrs-api/tree/master/jaxrs-tck) will not have ts.jte and will also have more tests to run.

Reply:
Yes indeed, so the version of REST in master will look quite a bit different. Same will hold for Faces and maybe Authentication (a Maven based TCK for Authentication is on the roadmap).